### ISSUE TITLE
Dashboard filter not applied to newly added question after saving dashboard

### ISSUE THREAD

**Issue by andreychirikba** [metabase/metabase#13607]
**Describe the bug**
When adding a new question to a dashboard, binding a dashboard filter to its field and saving the dashboard, then the filter is not applied to the question right away, but only after the page is manually reloaded.

**To Reproduce**
1. Create a dashboard and add a (date) filter to it.
2. Create a question, save it and add it to the dashboard when prompted.
3. Bind the dashboard filter to the newly added question.
4. Save the dashboard.
5. Notice the question is still showing unfiltered data.
6. Reload the page manually - only now is the filter applied to the question correctly.

**Expected behavior**
The filter should apply to the question right after it is bound to a field and the dashboard is saved.

**Information about your Metabase Installation:**
```
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1; Windows for Workgroups 3.11) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "13.0.2+8",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "13.0.2",
    "java.vm.name": "Eclipse OpenJ9 VM",
    "java.vm.version": "openj9-0.18.0",
    "os.name": "Linux",
    "os.version": "4.15.0-66-generic",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "mysql"
    ],
    "hosting-env": "unknown",
    "application-database": "mysql",
    "application-database-details": {
      "database": {
        "name": "MySQL",
        "version": "5.7.32"
      },
      "jdbc-driver": {
        "name": "MariaDB Connector/J",
        "version": "2.6.2"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-10-22",
      "tag": "v0.36.8",
      "branch": "release-0.36.x",
      "hash": "e49cdb1"
    },
    "settings": {
      "report-timezone": "Europe/Brussels"
    }
  }
}
```

**Severity**
A nuisance.

**Additional context**
Not a new one, has been present in all previous versions that I can remember (0.34+).

**Comment by flamber** [comment]
Duplicate of #5332

**Comment by flamber** [comment]
Your issue is fixed on 0.37.0.2
